unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/EmailPromptHUD/EmailPromptHUDController.cs

```diff
@@ -31,6 +31,7 @@ public void SetVisibility(bool visible)
             Utils.UnlockCursor();
             view.gameObject.SetActive(true);
             view.showHideAnimator.Show();
+            WebInterface.ReportAnalyticsEvent("open email popup");
```

Comment:
Sounds like we should put all these strings as constants under a common static class.

Reply:
Really? This are like super specific strings... I can understand if you want them as a constant in that class, but I fail to see the porpoise on having a common static class for this 🤔